### DISCUSSION TITLE
![image](https://user-images.githubusercontent.com/59680788/162798908-cfff0d5c-1212-402f-8582-e617a14bbf66.png)

### DISCUSSION THREAD

**Discussion by wisdomInterconnect** [prisma/prisma#17427] [Q&A]
![162798908-cfff0d5c-1212-402f-8582-e617a14bbf66](https://user-images.githubusercontent.com/13049676/213671380-40b01232-0e3e-4b28-9b2e-363bbf970f35.png)
whenever type any prisma command. getting this error.

_Originally posted by @srinivassunkari in https://github.com/prisma/prisma/issues/2660#issuecomment-1095348798_

**Comment by wisdomInterconnect** [comment]
what the ideal way to solve the problem

**Reply by nurul3101** [reply]
Hey @wisdomInterconnect 👋 
Which version of Prisma are you using? Are you invoking this command behind a corporate network firewall?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
